### PR TITLE
comment some useless tests until migration is finish

### PR DESCRIPTION
**Pull request by renatonlima**

### CODE REVIEW

**Comment by mileo** [comment]
LGTM
:+1:

**Comment by mbcosta** [comment]
:+1:

**Comment by sadamo** [comment]
:+1:

**Comment by rvalyi** [comment]
:+1: good catch!

File: l10n_br_account/test/account_supplier_invoice.yml [hunk 1]
```diff
@@ -45,14 +45,3 @@
 -
   !assert {model: account.invoice, id: account_invoice_supplier0}:
     - state == 'draft'
```
Comment (danimaribeiro) [inline l10n_br_account/test/account_supplier_invoice.yml:47]: Do the same here then:
- fiscal_position.id == ref('fiscal_position_sale')

**Comment by rvalyi** [comment]
@danimaribeiro ok

So folks after that we rebase all the PR's and many of them will be green and read for your validation before merge; finally...

**Comment by rvalyi** [comment]
@danimaribeiro I did what you suggested. Merging now as this one already passed https://travis-ci.org/akretion/l10n-brazil

**Pull request merged**